sympy/logic/boolalg.py

```diff
@@ -3086,7 +3089,7 @@ def _apply_patternbased_simplification(rv, patterns, measure,
     Rel, nonRel = sift(rv.args, lambda i: isinstance(i, Relational),
                        binary=True)
     if len(Rel) <= 1:
-        return rv
+        return simplify_logic(rv)
```

Comment:
We don't want to call `simplify_logic` here. It is an expensive heuristic function that should not be used during this pattern-based simplification procedure.